src/os/tempfile.go

```diff
@@ -17,7 +17,8 @@ var minrandMutex sync.Mutex
 func init() {
 	// Avoid getting same results on every run
 	now := time.Now()
-	seed := uint32(Getpid()) ^ uint32(now.Nanosecond()) ^ uint32(now.Unix())
+	// seed := uint32(Getpid()) ^ uint32(now.Nanosecond()) ^ uint32(now.Unix())
```

Comment:
To get this PR working I needed to adjust the seeding function. @aykevl is this still fine or should we increase entropy somehow?

Reply:
@aykevl what do you think?

Also @leongross you should probably remove the commented out line.

Reply:
Why didn't this work?

Reply:
I am not sure anymore what I thought the issue was there. Using the prior implementation seems to work now so I will revert this change.